package.json

```
@@ -36,9 +36,9 @@
   },
   "dependencies": {
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
-    "@edx/frontend-component-footer": "10.3.0",
-    "@edx/frontend-component-header": "2.6.1",
-    "@edx/frontend-platform": "1.15.6",
+    "@edx/frontend-component-footer": "11.1.1",
+    "@edx/frontend-component-header": "^3.1.1",
+    "@edx/frontend-platform": "2.5.0",
```

Running tests and build works with @edx/frontend-platform 2.5.0, I also tested with 2.7.0 and works without issues to. I think it may be more appropiate to bump the package to 2.7.0 so it's more updated.

```suggestion
    "@edx/frontend-platform": "2.7.0",
```